admin/class-admin-apple-preview.php

```diff
@@ -121,25 +62,22 @@ public function get_preview_html( $theme = null, $add_settings = false ) {
 					$export->format_byline( null, $author, $date )
 				);
 
-				// Get the order of the top components
-				$component_order = $settings->get( 'meta_component_order' );
-				foreach ( $component_order as $component ) {
+				// Get the order of the top components.
+				foreach ( $theme->get_value( 'meta_component_order' ) as $component ) {
```

Comment:
Is there any concern this function could ever return NOT an array (either via error or filtering) and cause an error. Though I suppose we weren't checking before.